packages/react-dom/src/client/ReactDOM.js

```diff
@@ -841,6 +845,12 @@ const ReactDOM: Object = {
   unstable_createSyncRoot: createSyncRoot,
   unstable_flushControlled: flushControlled,
 
+  unstable_scheduleHydration(target: Node) {
+    if (target) {
```

Comment:
Mostly because it's pretty common to pass in getElementById and probably not worth taking down the site since this feature has no important semantic meaning other than perf. It should probably be a warning at some point though.

Reply:
Yeah, seems like we should at least warn since the overall scheduling behavior is a bit opaque to begin with. (Might be easy to overlook an error like a misspelled id.)